ibm/mas_devops/common_vars/casebundles/v9-240920-amd64.yml

```diff
@@ -0,0 +1,100 @@
+---
+# Case bundle configuration for IBM Maximo Operator Catalog 240920
+# -----------------------------------------------------------------------------
+# In the future this won't be necessary as we'll be able to mirror from the
+# catalog itself, but not everything in the catalog supports this yet (including MAS)
+# so we need to use the CASE bundle mirror process still.
+
+catalog_digest: sha256:d07d86ac13208fd9af198658ce79a90d3820797d0ce8e43450d99797bfd5aeca
```

Comment:
What catalog digest is it referring to? The one used in dark release? Or the https://github.ibm.com/maximoappsuite/ibm-maximo-operator-catalog/tree/feature-channel ?